examples/HP34401A_measure_currents.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+#    HP34401A_measure_currents.py - functions with PyVisa to monitor currents
+#      using a HP34401A multimeter. It supposes that there is only one GPIB
+#      instrument (the HP34401A multimeter) plugged to the computer.
+#      To use this example just put the example in the directory where you
+#      call python then just run:
+#      $> python
+#      >>> from HP34401A_measure_currents.py import *
```

Comment:
This types of import are usually not a good idea. I you are showcasing a command line utility, put countMeasA withint the main and call: `python myscript.py`